sys/Makefile.include

```diff
@@ -89,3 +89,9 @@ endif
 ifneq (native,$(BOARD))
   INCLUDES += -I$(RIOTBASE)/sys/libc/include
 endif
+
+ifneq (,$(filter puf_sram,$(USEMODULE)))
+  # allocate space for 3 x 32 Bit integers: seed, state, marker
+  export PUF_SRAM_LEN ?= 12
+  export LINKFLAGS += $(LINKFLAGPREFIX)--defsym=_puf_sram_len=$(PUF_SRAM_LEN)
```

Comment:
Is this necessary? The linker should not need to be told how much space the puf section needs.